*** org-ja.texi.orig	2011-12-10 13:52:25.000000000 +0900
--- org-ja.texi	2011-12-10 13:53:09.000000000 +0900
***************
*** 10,15 ****
--- 10,21 ----
  @setfilename ../../info/org-ja
  @settitle Org Mode マニュアル
  
+ @documentlanguage ja
+ @documentencoding utf-8
+ @iftex
+ @usedvipdfmx
+ @end iftex
+ 
  @set VERSION 7.5
  @set DATE March 2011
  
***************
*** 1301,1307 ****
  サブツリーの見出しを全て表示し、ひとつのサブツリーのためのコンテンツビ
  ューである。@orgcmd{C-c C-x b,org-tree-to-indirect-buffer}　間接的なバッファ
  @footnote{間接的なバッファとは、
! @ifinfo (@pxref{Indirect Buffers,,,emacs,GNU Emacs Manual})  @end ifinfo @ifnotinfo (see the Emacs manual for more information about indirect buffers)  @end ifnotinfo は全てのバッファを含んでいるが、カレントのツリーに制限されるだろう。間接的なバッファを編集することは、オリジナルのバッファに変更を加えることでもある。だがそのバッファの中での表示に影響を与えることはできない。}.
  の中にあるカレントのサブツリーを表示する。Nという数値付きの前置引数を
  つけると、N段階上の階層に上がるがそのツリーを捉える。もしもNがマイナス
  の値ならば、多くの階層まで遡る。@kbd{C-u}の前置引数をつけたならば、それ以
--- 1307,1317 ----
  サブツリーの見出しを全て表示し、ひとつのサブツリーのためのコンテンツビ
  ューである。@orgcmd{C-c C-x b,org-tree-to-indirect-buffer}　間接的なバッファ
  @footnote{間接的なバッファとは、
! @ifinfo (@pxref{Indirect Buffers,,,emacs,GNU Emacs Manual})
! @end ifinfo
! @ifnotinfo (see the Emacs manual for more information about indirect buffers)
! @end ifnotinfo
! は全てのバッファを含んでいるが、カレントのツリーに制限されるだろう。間接的なバッファを編集することは、オリジナルのバッファに変更を加えることでもある。だがそのバッファの中での表示に影響を与えることはできない。}.
  の中にあるカレントのサブツリーを表示する。Nという数値付きの前置引数を
  つけると、N段階上の階層に上がるがそのツリーを捉える。もしもNがマイナス
  の値ならば、多くの階層まで遡る。@kbd{C-u}の前置引数をつけたならば、それ以
