*** org-ja.texi	2011-09-02 00:08:14.000000000 +0900
--- org-ja.texi.fix	2011-09-02 00:08:58.000000000 +0900
***************
*** 1288,1294 ****
  a prefix argument show, on each level, all sibling headings.  With double
  prefix arg, also show the entire subtree of the parent.  @orgcmd{C-c C-k,show-branches} Expose all the headings of the subtree, CONTENT view for
  just one subtree.  @orgcmd{C-c C-x b,org-tree-to-indirect-buffer} Show the
! current subtree in an indirect buffer@footnote{The indirect buffer @ifinfo (@pxref{Indirect Buffers,,,emacs,GNU Emacs Manual})  @end ifinfo @ifnotinfo (see the Emacs manual for more information about indirect buffers)  @end ifnotinfo will contain the entire buffer, but will be narrowed to the current tree.  Editing the indirect buffer will also change the original buffer, but without affecting visibility in that buffer.}.  With a numeric
  prefix argument N, go up to level N and then take that tree.  If N is
  negative then go up that many levels.  With a @kbd{C-u} prefix, do not
  remove the previously used indirect buffer.
--- 1288,1299 ----
  a prefix argument show, on each level, all sibling headings.  With double
  prefix arg, also show the entire subtree of the parent.  @orgcmd{C-c C-k,show-branches} Expose all the headings of the subtree, CONTENT view for
  just one subtree.  @orgcmd{C-c C-x b,org-tree-to-indirect-buffer} Show the
! current subtree in an indirect buffer@footnote{The indirect buffer
! @ifinfo (@pxref{Indirect Buffers,,,emacs,GNU Emacs Manual})
! @end ifinfo
! @ifnotinfo (see the Emacs manual for more information about indirect buffers)
! @end ifnotinfo
! will contain the entire buffer, but will be narrowed to the current tree.  Editing the indirect buffer will also change the original buffer, but without affecting visibility in that buffer.}.  With a numeric
  prefix argument N, go up to level N and then take that tree.  If N is
  negative then go up that many levels.  With a @kbd{C-u} prefix, do not
  remove the previously used indirect buffer.
