*** org-ja.texi	2011-04-21 01:20:09.000000000 +0900
--- org-ja.texi.fix	2011-04-21 01:20:54.000000000 +0900
***************
*** 1279,1287 ****
  C-k,show-branches} Expose all the headings of the subtree, CONTENT view for
  just one subtree.  @orgcmd{C-c C-x b,org-tree-to-indirect-buffer} Show the
  current subtree in an indirect buffer@footnote{The indirect buffer @ifinfo
! (@pxref{Indirect Buffers,,,emacs,GNU Emacs Manual})  @end ifinfo @ifnotinfo
! (see the Emacs manual for more information about indirect buffers)  @end
! ifnotinfo will contain the entire buffer, but will be narrowed to the
  current tree.  Editing the indirect buffer will also change the original
  buffer, but without affecting visibility in that buffer.}.  With a numeric
  prefix argument N, go up to level N and then take that tree.  If N is
--- 1279,1290 ----
  C-k,show-branches} Expose all the headings of the subtree, CONTENT view for
  just one subtree.  @orgcmd{C-c C-x b,org-tree-to-indirect-buffer} Show the
  current subtree in an indirect buffer@footnote{The indirect buffer @ifinfo
! (@pxref{Indirect Buffers,,,emacs,GNU Emacs Manual})
! @end ifinfo
! @ifnotinfo
! (see the Emacs manual for more information about indirect buffers)
! @end ifnotinfo
! will contain the entire buffer, but will be narrowed to the
  current tree.  Editing the indirect buffer will also change the original
  buffer, but without affecting visibility in that buffer.}.  With a numeric
  prefix argument N, go up to level N and then take that tree.  If N is

